setup.cfg

```diff
@@ -62,7 +62,9 @@ gpiozero_mock_pin_classes =
     mocktriggerpin   = gpiozero.pins.mock:MockTriggerPin
 
 [tool:pytest]
-addopts = -rsx --cov --tb=short
+# addopts = -rsx --cov --tb=short
+# Uncomment the above line to provide basic coverage information in local runs
```

Comment:
I assume that this was enabled for a reason, so perhaps it should be left enabled by default? :shrug: